SparkPost/2web2ui
# SA-1877: Add Segment tracking for Blocklist remediation steps

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed

- Added onClick handler for <SupportTicketLink>
- Added Segment constants
- Added Segment Tracking
- Added tests

### How To Test

- Go to /signals/blocklist/incidents/
- Click on an non-spamhaus incident.
- Go to https://app.segment.com/sparkpost-daniel-chalef/sources/webui_dev/debugger
- Click the action button in remediation steps
- Check that a segment event happened
- Now go back a page and select a spamhaus incident.
- Click 'Contact Support'
- Verify a segment event happened
- Click `Learn More`
- Verify a segment event happened

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
